server/db/050-insertColumns.sql

```diff
@@ -0,0 +1,22 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 50;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE clients ADD COLUMN Organization text ;
```

Comment:
Please add these columns to the `clients_extension` table, not `clients`

++ Please label them in all lowercase like so:

organization
funder
postal_code

to match the other columns!

Reply:
Ok! Changes made!

Reply:
done!